truss/cli/train/metrics_watcher.py

```diff
@@ -45,15 +46,59 @@ def _format_bytes(self, bytes_val: float) -> Tuple[str, str]:
             return f"{bytes_val / (1024 * 1024 * 1024):.2f} GB", color_map[unit]
         return f"{bytes_val:.2f} bytes", color_map[unit]
 
+    def _format_storage_utilization(self, utilization: float) -> Tuple[str, str]:
+        percent = round(utilization * 100, 4)
+        if percent > 90:
+            return f"{percent}%", "red"
+        elif percent > 70:
+            return f"{percent}%", "yellow"
+        return f"{percent}%", "green"
+
     def _get_latest_metric(self, metrics: List[Dict]) -> Optional[float]:
         """Get the most recent metric value"""
         if not metrics:
             return None
         return metrics[-1].get("value")
 
-    def create_metrics_table(self, metrics_data: Dict) -> Table:
+    def _get_latest_storage_metrics(
+        self, storage_data: Optional[Dict[str, List[Dict]]]
+    ) -> Optional[Tuple[int, float]]:
+        if not storage_data:
+            return None
+        usage_data = storage_data.get("usage_bytes")
+        utilization_data = storage_data.get("utilization")
+        if not usage_data or not utilization_data:
+            return None
+        usage_value = usage_data[-1].get("value", None)
+        utilization_value = utilization_data[-1].get("value", None)
+        if not usage_value or not utilization_value:
+            return None
+        return cast(int, usage_value), cast(float, utilization_value)
+
+    def _maybe_format_storage_table_row(
+        self, table: Table, label: str, storage_data: Optional[Dict[str, List[Dict]]]
+    ) -> bool:
+        if not storage_data:
+            return False
+        maybe_values = self._get_latest_storage_metrics(storage_data)
+        if not maybe_values:
+            return False
+        raw_usage, raw_utilization = maybe_values
+        usage_value, usage_color = self._format_bytes(raw_usage)
+        utilization_value, utilization_color = self._format_storage_utilization(
+            raw_utilization
+        )
+        table.add_row(
+            label,
+            Text(usage_value, style=usage_color),
+            Text(utilization_value, style=utilization_color),
+        )
+        return True
+
+    def create_metrics_table(self, metrics_data: Dict) -> Columns:
         """Create a Rich table with the metrics"""
-        table = Table(title="Training Job Metrics")
+        tables = []
+        table = Table(title="Compute Metrics")
```

Comment:
nit: This is getting a bit big, how about `_create_metrics_table` and `_create_storage_metrics_table` helpers?